build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+#Cleanup old server/build files
+#Remove the generated folder called
+rm -r ~/"server" 2>/dev/null
```

Comment:
You don't need the quotes around `server` here. You would if that name had, for example, a space in it, but when it's a simple name like this you don't. It's not wrong, but it's also not necessary.